.codecov.yml

```diff
@@ -0,0 +1,16 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  status:                        # this controls whether a pull request will be blocked by a decrease in coverage
+    # Learn more at https://docs.codecov.com/docs/commit-status
+    project:
+      default:
+        threshold: 5%            # allows coverage to drop by up to the percent noted and still post a success status
+        #informational: true     # If true is specified the resulting status will pass no matter what the coverage is or what other settings are specified
+    patch: off
+
+ignore:
+  - "node"  # ignore folders and all its contents
+  - "runtime"  # ignore folders and all its contents
+  - "inspect"  # ignore folders and all its contents
```

Comment:
we don't have this folder (actually what does this file do?)

Reply:
Removed. This is copied from Acala for test purpose.